### DISCUSSION TITLE
External links not working

### DISCUSSION THREAD

**Discussion by caitlindev** [vercel/next.js#19172] [Help]
I hardly ever have to ask questions like this, I usually find what I'm looking for by Googling, but this is a dead end for me. I took over an old NextJS site, hosting it on Netlify here: https://pedantic-franklin-a36c00.netlify.app/ and I can't get any of the external links to work, including a mailto link. There's the "Contact" link in the main nav, in the footer, and the "Crafted by Wirekite" link in the foot. I'm losing my mind here, any feedback would be great, thank you!

**Comment by kachkaev** [comment]
How do working and non-working links look in the code?

Possible solution: https://stackoverflow.com/questions/61059111/external-link-is-not-working-in-next-js-when-you-want-to-use-link-component

**Reply by caitlindev** [reply]
Thanks for the quick reply! I'm only using the <Link> component for internal links, and standard <a> tags for external links, as the internet told me to do...

Examples of _working_ internal links using <Link>:
<img width="641" alt="working_internal" src="https://user-images.githubusercontent.com/9014492/99153197-1200bf00-2675-11eb-9273-6f6c16d5f334.png">
<img width="538" alt="working_internal2" src="https://user-images.githubusercontent.com/9014492/99153198-12995580-2675-11eb-929b-3ee1e6d38b68.png">

Examples of _non-working_ external links using standard <a> tag:
<img width="1130" alt="Screen Shot 2020-11-14 at 12 30 16 PM" src="https://user-images.githubusercontent.com/9014492/99153284-7c196400-2675-11eb-8a9a-3bfe52c84ea0.png">
<img width="1021" alt="Screen Shot 2020-11-14 at 12 30 31 PM" src="https://user-images.githubusercontent.com/9014492/99153285-7cb1fa80-2675-11eb-830e-76798385be91.png">

**Reply by caitlindev** [reply]
Even clearing everything out of my homepage and just including the example from the possible solution you linked to STILL doesn't do anything upon click of the button!
<img width="929" alt="Screen Shot 2020-11-14 at 12 43 10 PM" src="https://user-images.githubusercontent.com/9014492/99153576-4d9c8880-2677-11eb-934b-ef7099d18324.png">

**Comment by caitlindev** [comment]
Resolved! There was an event listener on the document in their Header component with preventDefault() on it 🙄. Thanks for your time and attention @kachkaev
